### PR TITLE
Biqian/init

### PR DESCRIPTION
**Pull request by bjqian**
Add init script to create azure resources

### CODE REVIEW

File: Initialize/publish.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```
Comment (wanlwanl) [inline Initialize/publish.sh:2]: How about delete it?

File: Initialize/init.sh [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            echo "ERROR: Unknow argument '$key'" 1>&2
+            print_usage
+            exit -1
+    esac
+done
+throw_if_empty "subscription" $subscription
+throw_if_empty "prefix" $PREFIX
+throw_if_empty "location" $LOCATION    
+
+az account set -s $subscription
```
Comment (wanlwanl) [inline Initialize/init.sh:56]: add `az cloud set --name xxx` for Global/Mooncake/FireFax...?

File: Initialize/init.sh [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    access_key=$(az storage account keys list -n biqianperfsa --query [0].value -o tsv)
+    az keyvault secret set --vault-name $KEYVAULT -n  $KV_SA_ACCESS_KEY --value "$access_key" 
+    echo "storage account $STORAGE_ACCOUNT created."
+else 
+    echo "storage account $STORAGE_ACCOUNT already exists. Skip creating.."
+fi
+
+if  [[ -z $(az aks show --name $KUBERNETES_SEVICES -g $RESOURCE_GROUP 2>/dev/null) ]];then
+    echo "start to create kubernetes services $KUBERNETES_SEVICES. May cost several minutes, waiting..."
+    az aks create -n $KUBERNETES_SEVICES  --vm-set-type VirtualMachineScaleSets  --kubernetes-version 1.16.10  --enable-managed-identity -s Standard_B4ms --nodepool-name captain --generate-ssh-keys \
```
Comment (vwxyzh) [inline Initialize/init.sh:98]: use D serial VM

File: Initialize/init.sh [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+set -e 
+trap "exit" INT
+
+function print_usage {
+    cat <<EOF
+Command 
+    $(basename $0)
+Arguments
+   --prefix|-p                          [Requied] Used to distingush your perf resources with others'
+   --subscription|-s                    [Requied] The subscriton used to create resources
```
Comment (vwxyzh) [inline Initialize/init.sh:11]: optional

File: Initialize/init.sh (hunk 2 above)
Comment (bjqian) [reply Initialize/init.sh]: add cloud as optional

File: Initialize/publish.sh (hunk 1 above)
Comment (bjqian) [reply Initialize/publish.sh:2]: update next time

**Pull request merged**